CONTRIBUTING.md

```diff
@@ -1,14 +1,31 @@
+
+#### Contribute component design
+If you are looking to submit a newly designed component to this project we first suggest submitting an issue in our [issue tracking](https://github.com/OfficeDev/office-ui-fabric-react/issues) section of GitHub in order for our team and community to come to an agreement that the proposed component/feature makes sense for our project. We also suggest looking at our [Adobe XD Toolkit](https://static2.sharepointonline.com/files/fabric/fabric-website/files/officeuifabric.zip) to see if the component has not already been designed.
+
+If you are a Microsoft employee and want to contribute an internal component, please check the internal version of the [Adobe XD Toolkit (Microsoft Employees)](https://microsoft.sharepoint.com/teams/OfficeUIFabric97)
+or if you have more in depth questions regarding our design system please visit our internal Microsoft Teams [channel](https://teams.microsoft.com/l/channel/19%3a73a5dbc26c9a4d8d91264611995bbdbb%40thread.skype/Fabric%2520Design?groupId=ffe264f2-14d0-48b5-9384-64f808b81294&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47).
+
+#### Experimental components
+Experimental components are any new components that do not currently exist within our [Adobe XD Toolkit](https://static2.sharepointonline.com/files/fabric/fabric-website/files/officeuifabric.zip) and new components likely to undergo major API changes shortly after release. By Default, our `create-component` script will place the component folder and files in packages/experiments. new components stay in the experimental phase until a Fabric team member approves the migrations from Experimental to Fundamental. Experimental components are developed in our packages/experiments project.
+
+#### Fundamental components
+Fundamental components are the official React representation of our [Adobe XD Toolkit](https://static2.sharepointonline.com/files/fabric/fabric-website/files/officeuifabric.zip) and receive higher priority in respect to stabilization, bug fixes, accessibility, and general design resourcing. Components **cannot** start off as a Fundamental component as it will need some time to properly stabilize. A good portion of the components are currently integrated withing some of our major products. Fundamental components are worked on within our packages/office-ui-fabric-react folder.
+
+#### Creating a component via the command line
 1. Open a command prompt in the root directory of your project directory.
 2. Run `npm run create-component -- --name YourAwesomeNewComponentName`
   * Our components use the Pascal Case naming convention.
 
-### Creating a component manually.
+#### Creating a component manually.
 1. From the root of the project, travel to `packages/experiments/src/components/`
```

Comment:
"Travel" is an interesting word. Usually people say "Navigate". Is this a Github thing?